src/fu-engine.c

```diff
@@ -7151,6 +7151,11 @@ fu_engine_init(FuEngine *self)
 	g_hash_table_insert(self->compile_versions,
 			    g_strdup("org.freedesktop.fwupd"),
 			    g_strdup(VERSION));
+#ifdef FWUPD_EFI_VERSION
```

Comment:
I don't see a lot of value to compile version. I think runtime is all we need.

Reply:
Ya, agree.  I've dropped this bit.